### DISCUSSION TITLE
blog/istio-cni-deep-dive/

### DISCUSSION THREAD

**Discussion by giscus** [rootsongjc/rootsongjc.github.io#344] [Announcements]
# blog/istio-cni-deep-dive/

本文详细解释了 Istio CNI 插件的设计理念、实现方式以及如何通过 Ambient Mode 提升安全性和权限管理。

https://jimmysong.io/blog/istio-cni-deep-dive/

**Comment by V-know** [comment]
CentOS 8 以后，包括 Aliyun OS Linux3 之后的 OS版本使用nftables替换了 iptables 命令。使用 istio-init 初始化网络便会报错，使用 istio-cni 插件是否能解决这个问题呢？或者 Istio 是否还提供了其它的解决方案呢？谢谢

**Reply by V-know** [reply]
目前通过在 K8s 节点上执行下面的命令，加载 iptables 相关的内核模块来解决了上面的问题：
```shell
modprobe ip_tables
modprobe iptable_filter
```

**Reply by rootsongjc** [reply]
感谢你提供的信息，我没有在阿里云上测试过，我在 GKE 上使用 CNI 没有遇到问题。
